Sprint-3/todo-list/02-guide_to_modularize_code.md

```diff
@@ -0,0 +1,40 @@
+# Why modularize code?
+
+Modularizing code means breaking a program into smaller, self-contained pieces (modules), each responsible for a specific functionality. This approach offers several key advantages:
+
+- Reusability – You can use the same code in different parts of a project or in other projects.
+- Maintainability – It's easier to fix bugs or update features when code is organized into smaller parts.
+- Separation of Concerns – Each part of the code handles a specific task, keeping things clear and organized.
+- Team Collaboration – Multiple people can work on different modules at the same time without conflict.
+- Scalability – You can add new features more easily as the project grows.
+- Testing – Smaller modules are easier to test individually.
+- Readability – Modular code is easier to read, understand, and navigate.
+
```

Comment:
It's probably worth mentioning explicitly that when you `import` something it causes all of the top-level statements in that file to get run? Which is why defining functions is fine, but e.g. adding an event listener isn't?